src/Test/Perf/install.csx

```diff
@@ -0,0 +1,103 @@
+#load "util\assemblies.csx"
+#load "util\ngen.csx"
+
+using System;
+using System.Diagnostics;
+using System.IO;
+
+// If we're being #load'ed by uninstall.csx, set the "uninstall" flag.
+var uninstall = Environment.GetCommandLineArgs(1) == "uninstall.csx";
+var message = uninstall ? "Restoring previous copy of" : "Installing";
+
+// TODO: Use actual command line argument parser so we can have help text, etc...
+var sourceFolder = Args.Length == 1 ? Args[0] : @"C:\Roslyn\Binaries\Release";
```

Comment:
The string `c:\roslyn` is used across a number of files.  Should we make it a constant and import it?